### DISCUSSION TITLE
Chrome and Firefox soon to reach major version 100 ... and change versioning scheme in user agent header

### DISCUSSION THREAD

**Discussion by mkuthan** [nielsbasjes/yauaa#453] [Q&A] [answered]
Major version 100 is a big milestone for both Chrome and Firefox. It also has the potential to cause breakage on websites as we move from a two-digit to a three-digit version number.

https://web.dev/chrome-firefox-100/

Does it impact Yauaa parser or not?

**Comment by nielsbasjes** [comment, marked as answer]
It should not have any impact.

Even this works fine on my machine:

    - test:
        input:
          user_agent_string: 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123456.0.4758.102 Safari/537.36'
        expected:
          DeviceClass                          : 'Desktop'
          DeviceName                           : 'Linux Desktop'
          DeviceBrand                          : 'Unknown'
          DeviceCpu                            : 'Intel x86_64'
          DeviceCpuBits                        : '64'

          OperatingSystemClass                 : 'Desktop'
          OperatingSystemName                  : 'Linux'
          OperatingSystemVersion               : '??'
          OperatingSystemVersionMajor          : '??'
          OperatingSystemNameVersion           : 'Linux ??'
          OperatingSystemNameVersionMajor      : 'Linux ??'
          OperatingSystemVersionBuild          : '??'

          LayoutEngineClass                    : 'Browser'
          LayoutEngineName                     : 'Blink'
          LayoutEngineVersion                  : '123456.0'
          LayoutEngineVersionMajor             : '123456'
          LayoutEngineNameVersion              : 'Blink 123456.0'
          LayoutEngineNameVersionMajor         : 'Blink 123456'

          AgentClass                           : 'Browser'
          AgentName                            : 'Chrome'
          AgentVersion                         : '123456.0.4758.102'
          AgentVersionMajor                    : '123456'
          AgentNameVersion                     : 'Chrome 123456.0.4758.102'
          AgentNameVersionMajor                : 'Chrome 123456'


https://try.yauaa.basjes.nl/?ua=Mozilla/5.0%20(X11;%20Linux%20x86_64)%20AppleWebKit/537.36%20(KHTML,%20like%20Gecko)%20Chrome/123456.0.4758.102%20Safari/537.36
